llvm/test/Transforms/Inline/promote-call-bitwidth.ll

```diff
@@ -0,0 +1,46 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
+; RUN: opt -S -passes=inline < %s | FileCheck %s
+target datalayout = "e-m:e-p200:128:128:128:64-p:64:64-i64:64-i128:128-n64-S128-A200-P200-G200"
+target triple = "riscv64-unknown-freebsd13.0"
```

Comment:
Does the triple matter at all here? The DataLayout doesn't correspond to anything real for RISC-V (it doesn't quite match a CHERI-RISC-V one), and I don't think we normally need the triple in these tests? (And if we do, drop the vendor and OS at least, they definitely don't matter)

Reply:
Done